src/styles/index.scss

```diff
@@ -1,3 +1,140 @@
 body {
   margin: 0;
 }
+
+$bodyFont: Roboto, sans-serif;
+$bodyColor: rgba(6, 11, 53, 1);
+$buttonColor: rgba(0, 172, 220, 1);
+$padingCard: 0 17px;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 17px;
+  padding-top: 33px;
+  padding-bottom: 18px;
+
+  width: 200px;
+
+  &__img {
+    height: 134px;
+    width: 160px;
+
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 30px;
+  }
+
+  &__title {
+    justify-content: center;
+    font-family: $bodyFont;
+    color: $bodyColor;
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+
+    width: 100%;
+    margin-bottom: 4px;
+  }
+
+  &__subtitle {
+    align-self: flex-start;
+
+    margin: 0;
+    margin-bottom: 16px;
+    font-family: $bodyFont;
+    line-height: 14px;
+    font-size: 10px;
+    font-weight: 400;
+    color: rgba(97, 96, 112, 1);
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    font-family: $bodyFont;
+    line-height: 16px;
+    font-size: 14px;
+    font-weight: 700;
+    color: #fff;
+
+    margin: 0;
+    height: 40px;
+    width: 100%;
+    border-radius: 5px;
+    background-color: $buttonColor;
+  }
+
+  &__button:hover {
+    border: 1px solid;
+    color: $buttonColor;
+    background-color: #fff;
+  }
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  width: 100%;
+  margin-bottom: 22px;
+
+  &__star-active {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background-image: url('../images/star-active.svg');
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-image: url('../images/star.svg');
+  }
```

Comment:
Move stars styles to the separate file and implement the modifier for active stars